components/wpt-results.html

```diff
@@ -138,12 +138,20 @@
           testRuns: {
             type: Array,
           },
+          filteredDirectories: {
+            type: Array,
+            value: []
+          },
         };
       }
 
       async connectedCallback() {
```

Comment:
Got it, and for things like this, readers can look them up in the Polymer documentation, so maybe no need for function comment.